src/screen/ParathreadsScreen.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {View, FlatList, StyleSheet, Linking} from 'react-native';
+import globalStyles, {standardPadding} from 'src/styles';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import {LeaseInfo, useParathreads} from 'src/api/hooks/useParaThreads';
+import {useParachainsLeasePeriod} from 'src/api/hooks/useParachainsLeasePeriod';
+import {Spinner, Card, Text, Divider} from '@ui-kitten/components';
+import type {ParaId} from '@polkadot/types/interfaces';
+import {formatNumber} from '@polkadot/util';
+import {useParaEndpoints} from 'src/api/hooks/useParaEndpoints';
+import {useParathreadInfo} from 'src/api/hooks/useParathreadInfo';
+import Identicon from '@polkadot/reactnative-identicon';
+import Padder from 'presentational/Padder';
+import {Account} from 'layout/Account';
+import {EmptyView} from 'presentational/EmptyView';
+
+type ParathreadData = {
+  name?: string;
+  homepage?: string;
+};
+
+export function ParathreadsScreen() {
+  const {data: leasePeriod} = useParachainsLeasePeriod();
+  const {data: parathreads, isLoading} = useParathreads();
+
+  const toHomepage = (url?: string) => {
```

Comment:
Let's move this function out of this component and use it directly in `ParathreadItem` instead of passing it as a prop to all items